src/Html.php

```diff
@@ -691,7 +691,7 @@ public static function displayTitle($ref_pic_link = "", $ref_pic_text = "", $ref
         }
 
         if ($ref_title != "") {
-            echo "<span class='btn bg-blue-lt pe-none' aria-disabled='true'>
+            echo "<span class='btn bg-blue-lt' aria-disabled='true'>
```

Comment:
Should be removed too.
```suggestion
            echo "<span class='btn bg-blue-lt'>
```